docs/README.md

```diff
@@ -0,0 +1,50 @@
+# lookout
+
+[lookout DesignDoc](https://docs.google.com/document/d/1pqz-_SHO5BsJE-aa8o_bAY3r5vR67amnWN8-qZc2UgY/edit#)
+
+## Components
+
+Lookout consists of next components
+ * **Server**
+   Main component, runing in a separate process.
+   Responsible for orchestrating all other services in response to review requests by external code review system and posting the results back.
```

Comment:
Suggestion to break this long sentence:
Responsible for orchestrating all the other services. It receives review requests made by an external code review system, and posts the results back.